_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,131 @@ The foundational pieces for "Government as a service" vision are available and i
 - Build a platform for enterprise interoperability (API Store and Tell us once);
 - One GC vision and Omni Channel Digital Strategy (Any device, any platform, any partner).
 
-[The GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html), [the Policy](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) and [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601), with the [Mandatory Procedures for Enterprise Architecture Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) have shifted GC departments and agencies are increasingly using open standards (data formats and protocols) and existing open source software.
-Being agile and focusing on the needs of the user is the new normal and there growing collaboration via open source software communities within and outside the GC.
-Organizations have migrated massively to the cloud and are sharing/reusing software and code to configure their environments.
-GC communities of practices such as Open source software, Cloud and Agile host regular events for GC IT employees.
+The [GC Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html), the [Policy](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) and [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601), with the [Mandatory Procedures for Enterprise Architecture Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602) have shifted GC departments and agencies. They are increasingly using open standards (data formats and protocols) and open source software.
```

Comment:
One sentence per line please

Reply:
Oupsy! Fixed. Could we add a test to spot this?

Reply:
Maybe.. https://github.com/JoshuaKGoldberg/sentences-per-line